### PR TITLE
Feature/refactoring

### PR DESCRIPTION
**Pull request by antschum**
Refactor estimator and load function. Simplify exception part and update Params parameter.

### CODE REVIEW

File: augurpy/estimators.py [hunk 1]
```diff
@@ -4,17 +4,16 @@
...
-class Params(TypedDict, total=False):
+class Params:
     """Type signature for random forest and logistic regression parameters."""
 
-    n_estimators: int
-    max_depth: int
-    max_features: Union[Literal["auto"], Literal["log2"], Literal["sqrt"], int, float]
-    penalty: Union[Literal["l1"], Literal["l2"], Literal["elasticnet"], Literal["none"]]
+    n_estimators: int = 100
+    max_depth: Union[int, None] = None
```
Comment (Zethson) [inline augurpy/estimators.py]: Isn't Union[int, None] basically Optional[int]?

File: augurpy/read_load.py [hunk 2]
```diff
@@ -30,26 +30,17 @@ def load(
...
             try:
-                cell_type = meta[cell_type_col]
-                label = meta[label_col]
-            except (KeyError, TypeError):
-                print("Missing label and / or cell type column.")
-                raise
+                cell_type = input[cell_type_col]
+                label = input[label_col]
+            except KeyError:
+                print("[bold red]No column names matching cell_type_col and label_col.")
```
Comment (Zethson) [inline augurpy/read_load.py:39]: For the coloring to work we need

`from rich import print` at the top of the file :)

File: augurpy/read_load.py [hunk 3]
```diff
@@ -30,26 +30,17 @@ def load(
...
-                out = AnnData(X=input, obs={"cell_type": cell_type, "label": label})
+        label = input[label_col] if meta is None else meta[label_col]
+        cell_type = input[cell_type_col] if meta is None else meta[cell_type_col]
 
-        else:
-            out = AnnData(X=input, obs={"cell_type": cell_type, "label": label})
-
-    else:
-        raise Exception("Not valid input.")
+        out = AnnData(X=input, obs={"cell_type": cell_type, "label": label})
```
Comment (Zethson) [inline augurpy/read_load.py]: Not really happy with the name "out". Can we find something better?

**Review by Zethson — CHANGES_REQUESTED** [review]
3 minor comments. Thanks!

File: augurpy/estimators.py (hunk 1 above)
Comment (antschum) [reply augurpy/estimators.py]: Yes! Ill change it to optional

File: augurpy/read_load.py (hunk 3 above)
Comment (antschum) [reply augurpy/read_load.py]: How about adata? Changed it to adata

**Comment by Zethson** [comment]
Great, thanks!

**Pull request merged**